javalib/src/main/scala/scala/scalajs/js/typedarray/TypedArrayBufferBridge.scala

```diff
@@ -36,45 +41,45 @@ package scala.scalajs.js.typedarray
 import java.nio._
 
 private[typedarray] object TypedArrayBufferBridge {
```

Comment:
I've been thinking about this and from what you say, IIUC, we have no choice but to keep the interface and the implementation separate. However, if that is the case, why not use a `.java` file for the interface (rather than a non-machine readable list)?

That'll at least allow usage sites to benefit of type checking. (and IIUC the use case for separating the javalib is having more than one usage site).